src/valkeyassert.h

```diff
@@ -1,4 +1,4 @@
-/* redisassert.h -- Drop in replacements assert.h that prints the stack trace
+/* valkeyassert.h -- Drop in replacements assert.h that prints the stack trace
```

Comment:
Let's call this file `serverassert.h` instead. It defines the functions called `_serverAssert` and `_serverPanic`.

Reply:
+1